File: AndroidApps/P2_C_feetToMeter/app/src/main/java/com/example/feettometer/MainActivity.java
package com.example.feettometer;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;
import android.widget.Toast;

public class MainActivity extends AppCompatActivity {
    // Fields
    EditText input;
    TextView result;
    Button calculate;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        getViews();
        onCalculatedButtonClick();
    }

    // getting views
    private void getViews () {
        input.findViewById(R.id.eTxtInput);
        result.findViewById(R.id.txtResult);
        calculate.findViewById(R.id.btnCalculate);
    }

    // click listener
    private void onCalculatedButtonClick() {
        calculate.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View view) {
                calculate();
            }
        });
    }

    // calculate
    private void calculate() {
        Toast.makeText(this, "Go", Toast.LENGTH_LONG).show();
    }
}

File: AndroidApps/PFragmentTest1/app/src/main/java/com/jinco/pfragmenttest1/MainActivity.kt
package com.jinco.pfragmenttest1

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.fragment.app.commit
import com.jinco.pfragmenttest1.databinding.ActivityMainBinding

class MainActivity : AppCompatActivity() {
    lateinit var binding:ActivityMainBinding
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        binding = ActivityMainBinding.inflate(layoutInflater)
        setContentView(binding.root)

        supportFragmentManager.commit {
            add(R.id.frame, RunFragment())
        }


    }
}